docs/user_guide.md

```diff
@@ -328,17 +369,19 @@ the .hocon file of the frontman (the only agent that is connected to the client)
 ```
 
 All the above .hocon "allow" blocks can be combined in a single "allow" block. An example
-is given [here](https://github.com/cognizant-ai-lab/neuro-san/blob/main/neuro_san/registries/math_guy_passthrough.hocon#L54)
+is given in here [math_guy_passthrough.hocon](
+    https://github.com/cognizant-ai-lab/neuro-san/blob/main/neuro_san/registries/math_guy_passthrough.hocon#L54)
 
 For a full reference, please check the [neuro-san documentation](https://github.com/cognizant-ai-lab/neuro-san/blob/main/docs/agent_hocon_reference.md#allow)
 
 ## Toolbox
 
-The **Toolbox** is a flexible and extensible system for managing tools that can be used by agents. It simplifies the integration of **LangChain** and **custom-coded tools** in a agent network configuration.
+The **Toolbox** is a flexible and extensible system for managing tools that can be used by agents. It simplifies the
+integration of **LangChain** and **custom-coded tools** in a agent network configuration.
 
 ### Default tools in toolbox
 
-#### Langchain tools
+#### Langchain tools in toolbox
```

Comment:
Why is it adding `in toolbox`?

Reply:
Headers must have unique names or else links to headers will not work. We have 'Coded tools' elsewhere in the document, and I added 'in toolbox' to make 'Coded tools in toolbox' unique. Did the same to Langchain tools for consistency in that block.